test/Table_Tests/src/In_Memory/Column_Spec.enso

```diff
@@ -11,6 +11,8 @@ import Standard.Test.Extensions
 
 polyglot java import org.enso.table.data.column.storage.Storage as Java_Storage
 
+main = Test_Suite.run_main spec
+
```

Comment:
any reason for this?

Reply:
I prefer it to be on the top of the file, as then everything to the bottom is just test cases. Just a matter of taste, I'm not attached to this.

Reply:
I find at the bottom easier as then spot it :)

We should settle on one eventually - not an issue either way